internal/api/ui/login/external_provider_handler.go

```diff
@@ -600,6 +601,28 @@ func (l *Login) jwtProvider(ctx context.Context, identityProvider *query.IDPTemp
 	)
 }
 
+func (l *Login) ldapProvider(ctx context.Context, identityProvider *query.IDPTemplate) (*ldap.Provider, error) {
+	password, err := crypto.DecryptString(identityProvider.LDAPIDPTemplate.Password, l.idpConfigAlg)
+	if err != nil {
+		return nil, err
+	}
+	var opts []ldap.ProviderOpts
+	if !identityProvider.LDAPIDPTemplate.TLS {
+		opts = append(opts, ldap.Insecure())
+	}
```

Comment:
isn't there an option for the port and all the mapping attributes as well?

Reply:
you're right, adding all custom opts